addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
I thought that the requestAnimationFrame because scroll events are asynchronous and don't fire immediately after setting the scroll 🤔